lib/kb_quast/baseclient.py

```diff
@@ -15,12 +15,12 @@
 try:
     from configparser import ConfigParser as _ConfigParser  # py 3
 except ImportError:
-    from ConfigParser import ConfigParser as _ConfigParser  # py 2
+    from configparser import ConfigParser as _ConfigParser  # py 2
```

Comment:
This seems absurd - if there's an import error try the exact same import again?

Reply:
See above